compiler/src/dotty/tools/dotc/sbt/ExtractAPI.scala

```diff
@@ -79,6 +79,9 @@ class ExtractAPI extends Phase {
       val pw = new PrintWriter(File(sourceFile.file.jpath).changeExtension("inc").toFile
         .bufferedWriter(append = true), true)
       try {
+        pw.println()
+        pw.println("API:")
+        pw.println("====")
```

Comment:
I assume this was not meant to stay there?

Reply:
This is me adding extra lines to a debug output , an improvement IMO that follows the format of ExtractDependencies

Reply:
still removed it